### PR TITLE
fix secondaries calc for photon and add theta calc to public funcions

### PR DESCRIPTION
**Pull request by MaxSac**

### CODE REVIEW

File: examples/Interaction.ipynb [hunk 1]
```diff
@@ -329,7 +329,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.2"
```
Comment (Jean1995) [inline examples/Interaction.ipynb:332]: It this a remnant?

Comment (MaxSac) [reply examples/Interaction.ipynb:332]: yes it's a remnant :sweat_smile:

**Review by Jean1995 — APPROVED** [review]

**Comment by sudojan** [comment]
Maybe this should also be described in the [docs](https://github.com/tudo-astroparticlephysics/PROPOSAL/blob/master/docs/config_docu.md)

**Comment by Jean1995** [comment]
> Maybe this should also be described in the [docs](https://github.com/tudo-astroparticlephysics/PROPOSAL/blob/master/docs/config_docu.md)

What exactly do you mean should be described in the config_docu? I think there is no json configuration so far, is it?

**Pull request merged**